src/components/modals/ScheduleFiltersDialog.tsx

```diff
@@ -83,7 +90,8 @@ export default function ScheduleFiltersDialog({
         setTab(index);
     };
 
-    const currentTabColor = tab === 0 ? LOCATIONS_COLOR[500] : CATEGORIES_COLOR[500];
+    const tabColors = [LOCATIONS_COLOR[500], CATEGORIES_COLOR[500], CLASS_TIME_COLOR[500]];
```

Comment:
Can move the `tabColors` constant out of the component, if you want to 🤗

Reply:
Was thinking the same thing, but are you thinking just out of the component, or out of the file? If the latter, where would it make sense to locate it?

Reply:
Within the same file is fine 🤗